scripts/run_clang_tidy_format.sh

```diff
@@ -37,12 +37,14 @@ echo Sources: ${SOURCES}
 rm -f tidy-*.fail
 
 # sh syntax is -c "string" [name [args ...]], so "tidy" here is the name and not included in "$@"
-echo ${HEADERS} ${SOURCES} | xargs -P${PARALLEL_JOBS} -n5 sh -c "${CLANG_TIDY} -export-fixes=\$(mktemp -p. tidy.fail-XXXX) \$@" tidy
-if [ $(find . -maxdepth 1 -name 'tidy.fail-*' -size +0 | wc -l) -gt 0 ] ; then
-	# clang-tidy put non-empty output in one of the tidy-*.fail files
-	cat tidy.fail-*
-	exit 1
-fi
+[ -n "${CHERIOT_SKIP_CLANG_TIDY-}" ] || {
```

Comment:
What is this about?  I think it might be accidentally included with this PR?

Reply:
It's the last commit (presently 6a19ddd); as the message says, `clang-tidy` takes forever but `clang-format` is fast, so it's nice to be able to run just the latter.  I can yank that commit out of this PR, though.

Reply:
I'm not sure how useful it is.  If you just want to run clang-format quickly, either, running clang-format-diff for the whole diff or just `clang-format -i` is easier.